server/src/main/java/com/objectcomputing/checkins/services/template_question/TemplateQuestionController.java

```diff
@@ -0,0 +1,140 @@
+package com.objectcomputing.checkins.services.template_question;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.*;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.netty.channel.EventLoopGroup;
+import io.reactivex.Single;
+import io.reactivex.schedulers.Schedulers;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import javax.annotation.Nullable;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ExecutorService;
+import java.util.stream.Collectors;
+
+@Controller("/services/feedback/questions")
```

Comment:
Our controller link could be more specific to differentiate between our schema for template questions and request questions

Reply:
Changed to template_questions instead of questions